ui/desktop/app/routes/scopes/scope/authenticate.js

```diff
@@ -28,13 +29,34 @@ export default class ScopesScopeAuthenticateRoute extends Route {
    * scope and all scopes (for org navigation).
    * @return {Promise} `{scope, scopes, authMethods}`
    */
-  model() {
+  async model() {
     const { id: scope_id } = this.modelFor('scopes.scope');
 
+    // Preload all authenticatable auth methods into the store
+    const authMethodsForAllScopes = await this.resourceFilterStore.queryBy(
+      'auth-method',
+      {
+        authorized_actions: [{ contains: 'authenticate' }],
+      },
+      {
+        scope_id: 'global',
+        recursive: true,
+      },
+    );
+    const scopeIDs = new Set(authMethodsForAllScopes.map((i) => i.scopeID));
+    // Fetch org scopes
+    // and filter out any that have no auth methods
+    const scopes = this.modelFor('scopes').filter(({ id: scope_id }) =>
+      scopeIDs.has(scope_id),
+    );
+    const authMethods = authMethodsForAllScopes.filter(
+      (i) => i.scopeID === scope_id,
```

Comment:
Can we call the `i` something like `authMethod` just to have descriptive variables? Same with the other map